src/config/PrinterList.json

```diff
@@ -1 +1,7 @@
-["g216bc", "h105bc"]
+[
+  { "printerName": "g216bc", "printerType": "toshiba" },
+  { "printerName": "h105bc", "printerType": "toshiba" },
+  { "printerName": "stub", "printerType": "squix" },
+  { "printerName": "aPrinterName", "printerType": "squix" },
+  { "printerName": "ssrtubebc-sq1", "printerType": "squix" }
+]
```

Comment:
This is what I was explaining the other day about adding the printerType here. Other options are to the add the labelTemplateName here, but that is a variable that should be preferable stored in Deployment project config, and therefore a VUE_APP variable, which can't be access from the JSON config file. So options:
1. Leave it as above, where the `printerType` is fixed and won't change, so it is okay to be hard coded. Then based off the printer type, access the correct labelTemplateName in code, which can be stored in the deployment project as a variable. 
2. Replace `printerType` here with `labelTemplateName`, which would have to be hardcoded here. 
3. Something else that I can't think of

Reply:
@stevieing @JamesGlover  ^

Reply:
Let me know your thoughts. Happy to update to option 2

Reply:
Based on our discussion you no longer need this as print my barcode will handle this. You can just go back to the array of printers as we are currently using a single template. I suspect this will change if we start to use this for other purposes but we can cross that bridge when we come to it.